src/components/DataTypes/date-time-helpers.tsx

```diff
@@ -0,0 +1,24 @@
+import memoize = require('lodash/memoize');
+
+const pad = (n: number) => (n < 10 ? `0${n}` : n);
+
+export const getDefaultDate = (): string => {
```

Comment:
This could be changed to 
```
() => {
  const date = new Date()
  date.setHours(0,0,0,0)
  return date.toISOString()
}
```

Reply:
The original format this function was outputting is `2019-03-13T00:00`, this would change it to `2019-03-13T00:00:00Z`. It looks desirable, but that function is untested, so it's hard to tell. I'm okay moving it to the new format, same as the normalize function